packages/eslint-config-vtex/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Block Typescript `enum`s.
```

Comment:
Isn't that a `changed`?

Reply:
I don't know 🤔 I actually added something for this hahahaha

Reply:
Mixed feelings but it's really a nitpick.